SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -562,6 +571,9 @@ void USpatialActorChannel::OnReserveEntityIdResponse(const Worker_ReserveEntityI
   
 	EntityId = Op.entity_id;
 	RegisterEntityId(EntityId);
+
+	// Register Actor with package map since we know what the entity id is.
+	NetDriver->PackageMap ->ResolveEntityActor(Actor, EntityId, improbable::CreateOffsetMapFromActor(Actor));
```

Comment:
This is a different PR right?

Reply:
Yes, but that PR was merged into this one.